deploy/penny-discord-bot-stack.yml

```diff
@@ -452,7 +452,8 @@ Resources:
       Environment:
         Variables:
           BOT_TOKEN_ARN: arn:aws:secretsmanager:eu-west-1:177420307256:secret:prod/penny/penny-bot/discord-bot-token-VmhBXH
-          # TODO: Add client id and secret
+          GH_CLIENT_SECRET_ARN: arn:aws:secretsmanager:eu-west-1:177420307256:secret:prod/penny/penny-bot/github-oauth-client-secret-hQSGeh
```

Comment:
Need to add the ARN to the permissions in the file too (they were around line 200 i think ... you already modified them once, a few commits ago)